contrib/clojure-package/test/good-test-ndarray.clj

```diff
@@ -35,3 +35,4 @@
      ndarray-or-double-or-float
      #{"org.apache.mxnet.MX_PRIMITIVES$MX_PRIMITIVE_TYPE"
        "org.apache.mxnet.NDArray"})))))
+
```

Comment:
#nitpick do we need this extra line?

Reply:
It's in the generator - not needed but we can tackle that in another PR touching that area.